jobs/experiment_main_summary_view.sh

```diff
@@ -1,17 +1,24 @@
 #!/bin/bash
 
 if [[ -z "$bucket" || -z "$date" ]]; then
-   echo "Missing arguments!" 1>&2
-   exit 1
+  echo "Missing arguments!" 1>&2
+  exit 1
+fi
+
+if [[ -z "$branch" ]]; then
+  branch=master
 fi
 
 git clone https://github.com/mozilla/telemetry-batch-view.git
 cd telemetry-batch-view
-sbt assembly
-spark-submit --master yarn \
-             --deploy-mode client \
-             --class com.mozilla.telemetry.views.ExperimentSummaryView \
-             target/scala-2.11/telemetry-batch-view-1.1.jar \
-             --bucket $bucket \
-             --from $date \
-             --to $date
+git checkout $branch
+if [ $? -eq 0 ]; then
+   sbt assembly
+   spark-submit --master yarn \
+                --deploy-mode client \
+                --class com.mozilla.telemetry.views.ExperimentSummaryView \
+                target/scala-2.11/telemetry-batch-view-1.1.jar \
+                --bucket $bucket \
```

Comment:
You'd want to add the `inbucket` parameter here

(Here's where that option is defined: https://github.com/mozilla/telemetry-batch-view/blob/master/src/main/scala/com/mozilla/telemetry/views/ExperimentSummaryView.scala#L32)